### PR TITLE
style(frontend): Match feature items of about oisy modal to design

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

We want to implement the new Modal Layout in the application. Besides margin and spacing adjustments, the new Figma design for the AboutWhyOisyModal has Icons and Title on the same line. 

First step: Let's introduce a component that handles this and adjust the structure of the i18n accordingly. Additionally the links within the descriptions, should be blue.

Figma example:

image

# Changes

- Implement AboutFeatureItem
- Adjust i18n structure
- add link color

# Tests

before: 
image

after:

image

### CODE REVIEW

File: src/frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -703,12 +703,30 @@
...
+				},
+				"verifiable_credentials": {
+					"title": "Unlock Extra Features with Verifiable Credentials",
+					"description": "$oisy_short integrates with the Internet Computer's <a class='no-underline blue-link'rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/docs/current/developer-docs/identity/verifiable-credentials/overview\">verifiable credentials</a> platform. Access additional features like airdrops and exclusive functionalities by acquiring certain types of credentials, enhancing your overall experience."
+				},
+				"open_source": {
+					"title": "Trust in Transparency with Open Source",
+					"description": "$oisy_short is an <a class='no-underline blue-link'rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://github.com/dfinity/oisy-wallet\">open-source</a> project led by a dedicated team within the <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://dfinity.org/\">DFINITY foundation</a>, supported by over 200 world-leading cryptographers and engineers. Open sourcing ensures transparency, allowing the community to audit, contribute, and trust in the wallet's integrity."
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/i18n/en.json:728]:
```suggestion
					"description": "$oisy_short is an <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"$oisy_repo_url\">open-source</a> project led by a dedicated team within the <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://dfinity.org/\">DFINITY foundation</a>, supported by over 200 world-leading cryptographers and engineers. Open sourcing ensures transparency, allowing the community to audit, contribute, and trust in the wallet's integrity."
```

File: src/frontend/src/lib/i18n/en.json [hunk 2]
```diff
@@ -703,12 +703,30 @@
...
+				},
+				"network_custody": {
+					"title": "No more private key anxiety with Network Custody",
+					"description": "$oisy_name implements network custody, an innovative approach that leverages the Internet Computer's advanced cryptography to eliminate the need for you to handle private keys directly. This provides a secure and user-friendly alternative to traditional methods, aligning with the “Onchain is the New Online“ movement — a shift toward solutions that enhance security and resilience against cyber attacks by running on a distributed network."
+				},
+				"fully_on_chain": {
+					"title": "Tamper-Proof Security with a Fully On-Chain Wallet",
+					"description": "Not only are your keys secured, but the entire wallet application is stored <a class='no-underline blue-link'rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/capabilities\">fully on-chain</a> and served directly into your browser from the Internet Computer. This means the entire wallet benefits from a decentralized trust model, making it tamper-proof and highly secure."
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/i18n/en.json:716]:
```suggestion
					"description": "Not only are your keys secured, but the entire wallet application is stored <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/capabilities\">fully on-chain</a> and served directly into your browser from the Internet Computer. This means the entire wallet benefits from a decentralized trust model, making it tamper-proof and highly secure."
```

File: src/frontend/src/lib/i18n/en.json [hunk 3]
```diff
@@ -703,12 +703,30 @@
...
+				},
+				"cross_device": {
+					"title": "Access your wallet anytime from anywhere",
+					"description": "Using <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://identity.ic0.app/\">Internet Identity</a>, $oisy_short can easily be used across all devices you have linked to your private decentralized identity. This ensures seamless and secure access to your wallet whether you're on a smartphone, tablet, or desktop."
+				},
+				"verifiable_credentials": {
+					"title": "Unlock Extra Features with Verifiable Credentials",
+					"description": "$oisy_short integrates with the Internet Computer's <a class='no-underline blue-link'rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/docs/current/developer-docs/identity/verifiable-credentials/overview\">verifiable credentials</a> platform. Access additional features like airdrops and exclusive functionalities by acquiring certain types of credentials, enhancing your overall experience."
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/i18n/en.json:724]:
```suggestion
					"description": "$oisy_short integrates with the Internet Computer's <a class='no-underline blue-link' rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/docs/current/developer-docs/identity/verifiable-credentials/overview\">verifiable credentials</a> platform. Access additional features like airdrops and exclusive functionalities by acquiring certain types of credentials, enhancing your overall experience."
```

File: src/frontend/src/lib/styles/global/links.scss [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+
 a.no-underline {
 	&:hover,
 	&:focus {
 		text-decoration: none;
 	}
 }
+
+a.blue-link {
+	color: #{$brandeis-blue};
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/styles/global/links.scss:11]: is there a way to use tailwind directly in the hyperlink? or at least to use the color variables like `var(--color-brand-base)`?

File: src/frontend/src/lib/components/about/AboutWhyOisyModal.svelte
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/about/AboutWhyOisyModal.svelte:1]: i was thinking: is it maybe worth creating a list of items like `{title; description; icon}` and loop `AboutFeatureItem`? I don't remember if we have something against... @peterpeterparker , we discussed this once I think

File: src/frontend/src/lib/styles/global/links.scss (hunk 4 above)
Comment (loki344) [reply src/frontend/src/lib/styles/global/links.scss:11]: I tried, but it's not possible. I'm afraid to use color-brand-base for the reason that the link should always be blue, and if we change the brand color to yellow (as an example), the links would be yellow..

File: src/frontend/src/lib/components/about/AboutWhyOisyModal.svelte
Comment (loki344) [reply src/frontend/src/lib/components/about/AboutWhyOisyModal.svelte:1]: Yeah why not, will reduce amount of lines a bit

Comment (loki344) [reply src/frontend/src/lib/components/about/AboutWhyOisyModal.svelte:1]: Haha, nevermind - Actually same amount of code. But maybe makes it cleaner

**Review by AntonioVentilii — APPROVED** [review]
LGTM tks

**Pull request merged**